app/views/simple_smart_answers/_outcome.html.erb

```diff
@@ -1,4 +1,15 @@
-<div data-module="track-smart-answer" data-smart-answer-node-type="outcome" data-smart-answer-slug="<%= outcome.slug %>">
+<%
+  page_title = []
```

Comment:
I wonder if it would be worth creating a helper function that takes in a list of all the various strings that are included in the page title (error, outcome/question title, simple smart answer title) and spits out the formatted title? 

I just think the code would read a bit cleaner if we moved that logic outside the template, and instead of this:

```
<%
  page_title = []
  page_title << outcome.title
  page_title << @publication.title
  page_title << "GOV.UK"

  content_for :title do
    page_title.join(' - ')
  end
 %>
```
...we had something like this: 
```
<%
  content_for :title do
    some_page_title_builder_function(outcome.title, @publication.title)
  end
 %>
```

Reply:
Yes! Good idea - done 👍